.mergify.yml

```diff
@@ -434,3 +436,44 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}] {{ title }} (backport #{{ number }})"
+
+  - name: backport patches to all active minor branches
+    conditions:
+      - merged
+      - base=main
+      - label=backport-active-minors
+    actions:
+      backport:
+        assignees:
+          - "{{ author }}"
+        # NOTE: this list needs to be changed when a new minor branch is created
+        #       or an existing minor branch reached EOL.
+        branches:
+          - "9.0"
+          - "8.18"
+          - "8.17"
+          - "8.16"
```

Comment:
I find the naming a big confusing, e.g. `9.0` and `8.x` branches are treated within `backport-active-minors` but they are indeed 2 different majors.

Could we do something like
- `backport-active-8` : `8.16, 8.17, 8.18` 
- `backport-active-9`: `9.0` 
- `backport-active-all`: `9.0,  8.16, 8.17, 8.18, 7.17`

Reply:
That's a good question - I also had some thoughts about it when I raised this.

Let me ping you in the internal ticket, so we can agree with the naming all. and then I'll update this thread to circle back

Reply:
Agreed to move forward with your proposal